submissions/emmanuelEsuong/css/styles.css

```diff
@@ -80,6 +80,64 @@ ul {
 #signin{
     color:rgb(2, 209, 2);
 }
```

Comment:
Well done

Create a separate CSS file for the contact page. call it `contact.css` This file is already getting too long

Reply:
Updated. 

Thanks.